docs/tutorials/moving-pictures/moving-pictures.md

```diff
@@ -0,0 +1,336 @@
+
+## Running Qurro
+
+Since we'll be working with DEICODE output (i.e. feature loadings in a
+compositional biplot), we'll need to use the
+`qiime qurro unsupervised-rank-plot` command. (The `supervised-rank-plot`
+command is for working with feature differentials.)
+
+```
+qiime qurro unsupervised-rank-plot \
+    --i-table table.qza \
+    --i-ranks ordination.qza \
+    --m-sample-metadata-file sample-metadata.tsv \
+    --m-feature-metadata-file taxonomy.qza \
+    --o-visualization qurro-plot.qzv
+```
+
+##### Output Artifacts
+
+- `qurro-plot.qzv` [view](https://view.qiime2.org/?src=http%3A%2F%2Fbiocore.github.io%2Fqurro%2Ftutorials%2Fmoving-pictures%2Fdata%2Fqurro-plot.qzv) | [download](http://biocore.github.io/qurro/tutorials/moving-pictures/data/qurro-plot.qzv)
+
+You just generated your first Qurro plot! `qurro-plot.qzv` is a `.qzv` file --
+that is, a QIIME 2 visualization. You can view it either by running `qiime tools
+view qurro-plot.qzv` or by uploading the file to
+[view.qiime2.org](https://view.qiime2.org/). (You can also just click on the
+"view" link right above to see a precomputed version of this visualization.)
+
+## Interacting with a Qurro visualization
+
+Let's view `qurro-plot.qzv`, as described above.
+
+<img src="https://raw.githubusercontent.com/biocore/qurro/master/docs/tutorials/moving-pictures/screenshots/qurro1.png" alt="Qurro interface screenshot #1. No features are selected." />
+
+So right away we see two things: on the left a plot of rankings (in this case,
+loadings) for each feature, and on the right a plot of selected features' log
+ratios in samples. In this tutorial these plots will be referred to as the
+__rank plot__ and __sample plot__, respectively.
+
+Since no features are currently selected to be part of a log ratio, these plots
+look pretty empty. So let's select some features!
+
+### Selecting features to construct log ratios
+
+Recall that we'd like to analyze the log ratios between features' abundances.
+In Qurro, "selecting features" lets us define a __log ratio__ between multiple
+features' abundances in each sample. That is, we can select the numerator and
+denominator feature(s) that make up this log ratio.
+
+There are a few ways of selecting features in Qurro:
+
+- One way is just by clicking on the rank plot twice. The first click sets
+  the numerator feature for a log ratio, and the second click sets the
+  denominator feature for the log ratio.
+- You can also select features based on a text-search through their feature
+  IDs or metadata. For example, it's possible to construct the log ratio of all
+  features with taxonomy annotations containing the text `o__Fusobacteriales` over all
+  features with taxonomy annotations containing the text `o__Pseudomonadales`.
```

Comment:
This sentence is a bit confusing.

Reply:
`You can also select features based on a text-search through their feature IDs or metadata.` --> `You can also select features based on a **textual** search through their **IDs** or metadata.` Let me know if this is sufficient, or if you'd like more changes (I know this part is kinda awkward).